build.gradle

```diff
@@ -83,6 +83,7 @@ sourceSets {
 
 test {
     dependsOn(shadowJar)
+    dependsOn('dist')
```

Comment:
@smillst Shouldn't shadowJar already depend on dist? Otherwise, what does that target include in the fat jar?

Reply:
No, `dist` creates the dist directory and copies files from checker-framework to that directory.  `shadowJar` creates a fat jar using the Gradle dependencies; it doesn't use the dist directory.